compiler/circle-eval-diff/src/MetricPrinter.cpp

```diff
@@ -180,6 +180,111 @@ void MAEPrinter::dump(std::ostream &os) const
   }
 }
 
+// TODO Remove duplicate codes with MAEPrinter
```

Comment:
`MAEPrinter` -> `MAPEPrinter` ?

Reply:
There is duplicate code in MAEPrinter, so I added this comment.